### PR TITLE
Hotfix/v3.0.1

### PR DESCRIPTION
**Pull request by joelbcastillo**

### CODE REVIEW

**Comment by joelbcastillo** [comment]
This pull request **introduces 3 alerts** and **fixes 6** when merging be70114dc5f34e1943c1c99d666a828ba30ddd4b into 96e60866360c342da6e46fc960db5498ab0bb858 - [view on LGTM.com](https://lgtm.com/projects/g/CityOfNewYork/NYCOpenRecords/rev/pr-0a36e8742a2b79286e65c64245e1ca40d4f68c83)

**new alerts:**

* 1 for Unused named argument in formatting call
* 1 for Missing named arguments in formatting call
* 1 for Unused local variable

**fixed alerts:**

* 3 for Unused local variable
* 3 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by joelbcastillo** [comment]
This pull request **introduces 2 alerts** and **fixes 6** when merging dd386cef2e0dd145e7d9df1b38bc2d10fa857fa8 into 96e60866360c342da6e46fc960db5498ab0bb858 - [view on LGTM.com](https://lgtm.com/projects/g/CityOfNewYork/NYCOpenRecords/rev/pr-8c2fa9593e8b36be8d9f61ccec56cac3ac808136)

**new alerts:**

* 1 for Unused named argument in formatting call
* 1 for Missing named arguments in formatting call

**fixed alerts:**

* 3 for Unused local variable
* 3 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by joelbcastillo** [comment]
This pull request **introduces 2 alerts** and **fixes 6** when merging e4368451215773d307898d84f7726880fbb1e73b into 96e60866360c342da6e46fc960db5498ab0bb858 - [view on LGTM.com](https://lgtm.com/projects/g/CityOfNewYork/NYCOpenRecords/rev/pr-22771a5c5a18368650caa49171a296950f71ad17)

**new alerts:**

* 1 for Unused named argument in formatting call
* 1 for Missing named arguments in formatting call

**fixed alerts:**

* 3 for Unused local variable
* 3 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by joelbcastillo** [comment]
This pull request **introduces 2 alerts** and **fixes 6** when merging 7540ee1827f9b18eb9b37fe5c9330aced69e4235 into 96e60866360c342da6e46fc960db5498ab0bb858 - [view on LGTM.com](https://lgtm.com/projects/g/CityOfNewYork/NYCOpenRecords/rev/pr-31cdd9567a57ae20ecd5bf377880147bc405348d)

**new alerts:**

* 1 for Unused named argument in formatting call
* 1 for Missing named arguments in formatting call

**fixed alerts:**

* 3 for Unused local variable
* 3 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by joelbcastillo** [comment]
This pull request **introduces 2 alerts** and **fixes 6** when merging cfce2e1a08d34eead8d12fca0e6e924cd44359b0 into 96e60866360c342da6e46fc960db5498ab0bb858 - [view on LGTM.com](https://lgtm.com/projects/g/CityOfNewYork/NYCOpenRecords/rev/pr-870d75be5bda576eb1c39158dc4f8bf5bdb9a353)

**new alerts:**

* 1 for Unused named argument in formatting call
* 1 for Missing named arguments in formatting call

**fixed alerts:**

* 3 for Unused local variable
* 3 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by joelbcastillo** [comment]
This pull request **introduces 2 alerts** and **fixes 6** when merging ac49a7b0de2c9616af2ba28d3d07376e696cbcc7 into 96e60866360c342da6e46fc960db5498ab0bb858 - [view on LGTM.com](https://lgtm.com/projects/g/CityOfNewYork/NYCOpenRecords/rev/pr-5f6e092f345e8f61a2be949de1371b69e52303f6)

**new alerts:**

* 1 for Unused named argument in formatting call
* 1 for Missing named arguments in formatting call

**fixed alerts:**

* 3 for Unused local variable
* 3 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by joelbcastillo** [comment]
This pull request **introduces 2 alerts** and **fixes 6** when merging 079a436ed88d5dd4e2ec43d4339494b363abe56d into 1e74a8ff1843cd898a72183a7eb78deb110bb928 - [view on LGTM.com](https://lgtm.com/projects/g/CityOfNewYork/NYCOpenRecords/rev/pr-1023cc73a802541c77088f0b9772dc41db03e99e)

**new alerts:**

* 1 for Unused named argument in formatting call
* 1 for Missing named arguments in formatting call

**fixed alerts:**

* 3 for Unused local variable
* 3 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by joelbcastillo** [comment]
This pull request **introduces 2 alerts** and **fixes 6** when merging 47aab6078732c44a8b5ef3741eca8c2e86404201 into 1e74a8ff1843cd898a72183a7eb78deb110bb928 - [view on LGTM.com](https://lgtm.com/projects/g/CityOfNewYork/NYCOpenRecords/rev/pr-5a9a1001fef3bc1795653c32510094bc6b13bb5d)

**new alerts:**

* 1 for Unused named argument in formatting call
* 1 for Missing named arguments in formatting call

**fixed alerts:**

* 3 for Unused local variable
* 3 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by joelbcastillo** [comment]
This pull request **introduces 2 alerts** and **fixes 6** when merging 50cf6b7963036fac8adafce3192b01fccafd7b03 into d6c327234104ff3ea09585f29265699c7e74cedb - [view on LGTM.com](https://lgtm.com/projects/g/CityOfNewYork/NYCOpenRecords/rev/pr-06c49f86d363919cc0451eaf07bdd41f41b1e5c3)

**new alerts:**

* 1 for Unused named argument in formatting call
* 1 for Missing named arguments in formatting call

**fixed alerts:**

* 3 for Unused local variable
* 3 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: app/jobs.py [hunk 1]
```diff
@@ -20,11 +21,12 @@
 STATUSES_EMAIL_TEMPLATE = "email_templates/email_request_status_changed"
 
 
-@celery.task(autoretry_for=(OperationalError,), retry_kwargs={'max_retries': 5}, retry_backoff=True)
+@celery.task(autoretry_for=(OperationalError, SQLAlchemyError,), retry_kwargs={'max_retries': 5}, retry_backoff=True)
```
Comment (g-zhou) [inline app/jobs.py]: Should we add StatementError to `autoretry_for`?

File: openrecords.py [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+                CustomRequestForms,
+            ),
+        )
+    )
+
+    es_recreate()
+
+
+if __name__ == "__main__":
+    main(as_module=False)
```
Comment (g-zhou) [inline openrecords.py]: I think you can get rid of `as_module=False` as it's False by default.

File: app/jobs.py (hunk 1 above)
Comment (joelbcastillo) [reply app/jobs.py]: `StatementError` is a child of `SQLAlchemyError`. It should be caught by this.

File: openrecords.py (hunk 2 above)
Comment (joelbcastillo) [reply openrecords.py]: Updated.

**Comment by joelbcastillo** [comment]
This pull request **introduces 2 alerts** and **fixes 6** when merging e7ec5a08f10cb82c439656bdf21e2b63b0522aac into 5394aa7703429f9c1def1206108d464856fa09b2 - [view on LGTM.com](https://lgtm.com/projects/g/CityOfNewYork/NYCOpenRecords/rev/pr-3aec3cfc47e53202536d67782b87abfbd52fba8f)

**new alerts:**

* 1 for Unused named argument in formatting call
* 1 for Missing named arguments in formatting call

**fixed alerts:**

* 3 for Unused local variable
* 3 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by joelbcastillo** [comment]
This pull request **fixes 6 alerts** when merging 101166415bd87390171af80d1131868bfb5088ef into 5394aa7703429f9c1def1206108d464856fa09b2 - [view on LGTM.com](https://lgtm.com/projects/g/CityOfNewYork/NYCOpenRecords/rev/pr-4c168ec632b004cb83485d47bfab727b4b0fd293)

**fixed alerts:**

* 3 for Unused local variable
* 3 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by joelbcastillo** [comment]
This pull request **fixes 6 alerts** when merging 6ac1ce1811d21a8017ea4fdb3b57bf409b24f24d into 292cd8319d2ecac466ed44902dd98759007c4b2e - [view on LGTM.com](https://lgtm.com/projects/g/CityOfNewYork/NYCOpenRecords/rev/pr-ecf8cfeb057bda7d311e1bf84bc5f9fa7b9e0a8d)

**fixed alerts:**

* 3 for Unused local variable
* 3 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**